packages/syft-cli/setup.py

```diff
@@ -22,14 +23,14 @@
     author="OpenMined <info@openmined.org>",
     packages=find_packages(),
     install_requires=packages,
-    extras_require={"dev": dev_packages},
+    extras_require={"dev": dev_packages, "build": build_packages},
     include_package_data=True,
     entry_points={
         "console_scripts": [
-            "syft = cli:app",
-            "syft-cli = cli:app",
-            "syftcli = cli:app",
-            "syftctl = cli:app",
+            "syft = syft_cli.cli:app",
```

Comment:
We now have two monikers `packages/syft-cli` `packages/syft-cli/syft_cli`. Can we make this consistent?

Reply:
That is a great question, 
This is something, which took me long enough to also decide

This is mainly from the problem that the python modules 

do not allow hypens `-`

Why a separate package : 
One advantage for us to go for a a new module (like syft_cli inside syft-cli) packages .
We could easily separate out, all the commands apart from other config files (setup.py, documentation and several other files) when our project goes large.

Why use hypen in the initial package:

I think hypens, are only mainly improving the readability,most of the cli when publish on pypi 
have <tool_name>-cli.

I have looked several cli tools , most implement like this

repo name: typer-cli
module name: typer_cli
link: https://github.com/tiangolo/typer-cli

But I found azure cli,has a good structure for the same.

https://github.com/Azure/azure-cli